app/assets/javascripts/discourse/app/templates/user/notifications-index.hbs

```diff
@@ -8,6 +8,9 @@
   </div>
 {{/if}}
 
+{{notifications-filter value=filter onChange=(action (mut filter))}}
+<span class="user-notifications-filter-separator"></span>
```

Comment:
It would be better if we can position this using CSS without adding a separator element. How about wrapping the notifications-filter in a `<div>`, and then setting it to 100% width with a border-bottom.

Reply:
Sure, ill try this.

Reply:
Sure, ill try this out